docs/tenancy/1.x/identification-queue.md

```diff
@@ -86,12 +115,12 @@ class MailCustomer implements ShouldQueue
 {
     use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
 
-    /** @var Customer $customer */
+    /** @var Customer $tenant */
     public $tenant;
 
     public function __construct(Customer $customer, $tenant_key = null, string $tenant_identifier = null)
     {
-        $this->customer = $customer;
+        $this->tenant = $customer;
     }
```

Comment:
Just want to verify this section of code is "correct" for what its trying to display as I modified it and wasn't the author.

Reply:
Yeah, looks good to me, this was probably a mistake on my part